Contributors.md

```diff
@@ -1 +1,3 @@
-1. Dilpreet Singh (http://github.com/dilpreet96) : Open source contributor
```

Comment:
Hello @NishSatish,

You don't have to remove Dilpreet's name form the list, rather you only have to add your name on the list. Can you please revert this change, and add your name after Dilpreet's name ?

After you have made this change, please use ```git commit --amend``` and then **force** push your changes to send a pull request. You can also [squash](https://stackoverflow.com/questions/2563632/how-can-i-merge-two-commits-into-one) your commits after making the suggested changes if you are comfortable with git. 

Please let us know if you need any help, we'll be glad to assist you.

Reply:
I have completed the required changes. I then opened a pull request. I have commited the file again, please review it Sir @AbhilashG97 .

Reply:
Sir @AbhilashG97 can I restart the task?